### ISSUE TITLE
[YggTorrent] search broken

### ISSUE THREAD

**Issue by TheWiresharkGuy** [CravateRouge/qBittorrentSearchPlugins#10]
The search plugin **no longer works**, no results are returned 😢
(tested on various qBittorrent versions)

I suspect the new Cloudflare protection of YggTorrent to be the cause (JavaScript challenge I think).

Can you do something about it?

**Comment by CravateRouge** [comment]
Yep, you're right. When the plugin try to connect to YGGtorrent, the Cloudflare protection gives a JavaScript challenge and I don't see a lightweight solution (without an external module supporting JavaScript) :/

**Comment by TheWiresharkGuy** [comment]
Maybe a lead ...
 [https://github.com/Anorov/cloudflare-scrape](https://github.com/Anorov/cloudflare-scrape)

Too bad I don't have enough programming skills ^^

**Comment by CravateRouge** [comment]
The link is interesting but as I said, it needs NodeJS as a dependency to understand JavaScript. And this is not embedded in the default installation of python or qBittorrent so it makes the installation of the YggTorrent plugin heavier by adding an extra step.

**Comment by TheWiresharkGuy** [comment]
Looks like it's resolved.
I guess now it depends on whether or not the site admins activate the protection ...

**Comment by Hideman85** [comment]
Do you have an available fork using NodeJS for solving the challenge?

**Comment by CravateRouge** [comment]
I don't know an available fork using NodeJS but I think a dependency as a JavaScript engine is a bit heavy to just scrap the YggTorrent website. Futhermore, if the CloudFlare protection includes a Google Captcha, the JavaScript engine won't be enough. But if you know or made a fork, I will be glad to link it in the Readme.

**Comment by p060477** [comment]
pls update this yr plug in page : 

https://github.com/CravateRouge/qBittorrentSearchPlugins
Y ggTorrent

Download the plugin file or copy the following link.

Because YggTorrent requires your login info, this plugin requires a bit more work than most.
Firstly you'll need an account
cause the site really do not work at all..:
https://yggtorrent.ch/user/register
returns:
404 Not Found
